components/table/index.css

```diff
@@ -11,7 +11,7 @@
 
 .additional-row,
 .additional-row * {
-  max-height: 200px;
+  max-height: 500px;
```

Comment:
This is btw what I hate the most on the table component. This number has to be chosen very wisely otherwise the expand/collapse animation may look ugly.